File: fix_vue.js
const fs = require('fs');
const content = fs.readFileSync('web/src/App.vue', 'utf8');
let lines = content.split('\n');

function checkTags(start, end) {
  let depth = 0;
  for (let i = start; i <= end; i++) {
    const line = lines[i];
    if (line.includes('<div')) depth++;
    if (line.includes('</div')) depth--;
    console.log(`${i+1}: [depth: ${depth}] ${line}`);
  }
}

// MF AccordionItem is around 700 to 865
console.log("MF:");
checkTags(700, 865);
console.log("\nEquity:");
checkTags(1060, 1150);

File: fix_vue.py
with open('web/src/App.vue', 'r') as f:
    lines = f.readlines()

def check_tags(start, end, label):
    depth = 0
    print(f"--- {label} ---")
    for i in range(start, end):
        line = lines[i]
        if '<div' in line: depth += line.count('<div')
        if '</div' in line: depth -= line.count('</div')
        print(f"{i+1}: [depth: {depth}] {line.rstrip()}")
    print(f"Final depth: {depth}\n")

check_tags(700, 866, "MF")
check_tags(1060, 1152, "Equity")
